components/flags_ui/resources/brave_flags_overrides.js

```diff
@@ -20,4 +20,11 @@ document.addEventListener('DOMContentLoaded', function () {
       link.textContent = 'nightly channel'
     }
   })
+  document.querySelector('div.blurb-container').after(
+    (() => {
+      const p = document.createElement('p')
+      p.textContent = '* Overridden by active variations or a command line.'
```

Comment:
`brave://flags` is not localizable, all strings are hardcoded into html/js.
cc @brave/string-reviewers-team for grammar/style. The text is visible in the screenshot in PR description.